### ISSUE TITLE
Signup Emails: Magic link and Confirmation unnecessarily different

### ISSUE THREAD

**Issue by escapemanuele** [Automattic/wp-calypso#96287]
### Quick summary

![Image](https://github.com/user-attachments/assets/5db99484-5f3b-4099-a6c9-5403b6e2b715)

Left is the magic login email and right is the confirmation one.
Different size, fonts, etc...

### Steps to reproduce

- Create a new account and use the magic login link

### What you expected to happen

The emails to have a similar structure

### What actually happened

Emails were a lot different

### Impact

All

### Available workarounds?

There is no user impact

**Comment by vykes-mac** [comment]
This one should be a simple fix but design input is needed.

**Comment by lucasmendes-design** [comment]
Hi @vykes-mac, what do you need specifically? Is it only about the style or is it about the content as well?

**Comment by vykes-mac** [comment]
@lucasmendes-design I think what @escapemanuele is calling out is the styling difference, the different font-sizes, how the header is displayed and it seem one button stretches across and the other doesn't. I don't think it's about the content. @escapemanuele could you provide more info?

**Comment by escapemanuele** [comment]
Yeah @lucasmendes-design we encountered this together in Login/Signup walkthrough with Jacopo. It's about the different styles used!

**Comment by vykes-mac** [comment]
@lucasmendes-design Once you provide an update with the styling consistency I can make the changes.

**Comment by lucasmendes-design** [comment]
Great, the marketing team just shared with me the latest. Let me check and compare with this issue here.

**Comment by lucasmendes-design** [comment]
Update:

Figma file -> uwEavMdyNLecNt6Ru3M2Za-fi-2539_30900 

![Image](https://github.com/user-attachments/assets/f22f2880-f8c3-4a97-b5d2-cbf9143100ae)

**Comment by vykes-mac** [comment]
@lucasmendes-design I see  a new design for the "Confirmation email" but I don't see one for the "Magic link" email. this is what it looks like currently 

![Image](https://github.com/user-attachments/assets/37d041e0-fb89-4578-8eb3-1c0076dbd6ac)

**Comment by crisbusquets** [comment]
@Robertght, Lucas is on parental leave. Do you need help from the design-side? 👀

**Comment by supernovia** [comment]
@crisbusquets this came through Product Ambassadors again for triage. Do you know what team this should go to since Lucas is on parental leave?

**Comment by crisbusquets** [comment]
> Do you know what team this should go to since Lucas is on parental leave?

I can take care of this :)

To keep things consistent, let’s default to aligning all buttons to the left and use auto width so they adapt to the text length (so, the screenshot on the right from @escapemanuele's first message is the correct one).

**Comment by crisbusquets** [comment]
> [@crisbusquets](https://github.com/crisbusquets) Do we maintain the styling of the heading `Sign in to WordPress.com`? The confirmation email has part of the text bold, I'm not certain how we decide what part should be bold.

We should remove all the bold text in headings. The new proposal Lucas shared above omits it, because it felt somewhat random—We didn’t have a clear criteria for what should be bolded or not.

**Comment by vykes-mac** [comment]
Fix here https://github.a8c.com/Automattic/wpcom/pull/175648
